### ISSUE TITLE
MinimigAGA 1.2 core (wording): 68010 CPU, MHz missing

### ISSUE THREAD

**Issue by AmigaIstMiST** [rkrajnc/minimig-mist#42]
https://github.com/mist-devel/mist-binaries/issues/10

**Comment by AmigaIstMiST** [comment]
I would be helpful, if there would be more information about the cpu speed and how to configure it. I discovered by accident, that when enabling FASTRAM when using the AGA Chipset, the CPU is as fast as a 88Mhz 68020 CPU (what's quite impressive, there is even a video on yt where somebody plays DOOM using the 1.2 core on the MiST). My first impression was, that the speed increase only came from the faster RAM. Such a cool feature really shouldn't be hidden!

**Comment by rkrajnc** [comment]
Hi,

the CPU is actually always running at the same speed, what changes is the cache on/off state, and (faked) 16/32-bit access to chipRAM.

It's hard to put an exact number on the CPU speed, as different programs will report different speeds, and since the TG68 CPU core used in minimig is not cycle-exact with real 680x0 CPUs, you can't just say that in ECS mode it's a 7MHz 68000, and in AGA it's a 28MHz 68020, since it also depends on the enabled turbo mode.

But, there will be some changes to the CPU / cache in the near future, so maybe then I can change the OSD as well to something more descriptive.

**Comment by AmigaIstMiST** [comment]
Yes, it takes some time to realize, that the MinimigAGA is indeed new Amiga Hardware, and not a A500/A600/A1200 replica. I think it should somehow made more clear, that this is really some sort of "New Amiga" :-)
I think if the MinimgAGA has a TG68 CPU, then it should also be called a TG68 Cpu, not "68000" etc. A "TG68" is something newer, better, faster, thats compatible to the old CPUs. Just my opinion.
